assets/src/blocks/Happypoint/HappypointFrontend.js

```diff
@@ -0,0 +1,63 @@
+import { useHappypointImageData } from './useHappypointImageData';
+
+export const HappypointFrontend = ({
+  focus_image,
+  opacity,
+  mailing_list_iframe,
+  iframe_url,
+  id
+}) => {
+  const { imageData } = useHappypointImageData(id);
+
+  const {
+    background_src,
+    background_srcset,
+    background_alt,
+    background_sizes,
+    default_image,
+    engaging_network_id
+  } = imageData;
+
+  let url = '';
+  if (mailing_list_iframe || iframe_url) {
+    url = iframe_url || engaging_network_id;
+  }
+
+  const imgProps = {
+    src: background_src || default_image,
+    style: {
+      objectPosition: focus_image,
+      opacity: opacity ? (opacity / 100) : 0.3,
+    },
+    alt: background_alt
+  };
+
+  if (background_src) {
+    imgProps.srcSet = background_srcset || 'false';
+    imgProps.sizes = background_sizes || 'false';
+  }
+
+  return (
+    <section className="block block-footer block-wide happy-point-block-wrap">
+      <picture>
+        <img {...imgProps} />
+      </picture>
+      <div className="container">
+        <div className="row justify-content-md-center">
+          {mailing_list_iframe && url &&
```

Comment:
Looks great! One possibly dangerous issue: the old version renders the URL anyway if its set, despite the `mailing_list_iframe` being set or not, this is from the old Twig template:
 
`{% if ( fields.mailing_list_iframe == 'true' or fields.iframe_url ) %}`

That was wrong, the new version is correct according to the sidebar hint, but maybe we should:
- a) replicate the old behavior and create a new ticket to correct it for the next release
- b) flag the ticket to let editors know they have to ensure the `Use mailing list iframe` switch is ON...

Maybe @comzeradd can help deciding.

Reply:
Good catch @pablocubico! If we decide to keep this new behaviour we also probably need to update the documentation 🤔

Reply:
@mleray Let's go with option **a**. Better replicate the previous behavior and open a bug about the toggle to refine it.

I'd personally remove the toggle completely. If the url field has a values it should be enough. But let's discuss this after this is done.

Reply:
@comzeradd @pablocubico done! I'll open a ticket to investigate and improve the behaviour.

Reply:
Ticket created: https://jira.greenpeace.org/browse/PLANET-5489